### DISCUSSION TITLE
Starstruck achievement wrong count

### DISCUSSION THREAD

**Discussion by kuzmoyev** [community/community#20511] [Other Feature Feedback, Questions, & Ideas] [answered]
I've got 2x "Starstruck" achievements. But the details show that they are for the same repo for two different star counts.

<img width="451" alt="image" src="https://user-images.githubusercontent.com/15250762/178968761-e9bd99cd-d050-49f2-a52b-0d07e4a6a12e.png">

You can check it [here](https://github.com/kuzmoyev?achievement=starstruck&tab=achievements)

**Comment by kuzmoyev** [comment, marked as answer]
Oh, I'm an idiot. These are two levels of achievement. How do I delete the discussion? :)

**Reply by martin-braun** [reply]
You really don't want to delete this anyways. It helps other people who got confused by it as well, me included. Thanks.

**Reply by Geolim4** [reply]
Completely agree, asking is not being an idiot, it's wanting being less of one.

**Reply by BdR76** [reply]
I agree that just the `16` and `128` is somewhat confusing. I think the text should be `A repository by USER has collected this many stars` or maybe the next threshold stars should also be shown or something like that.

**Reply by kuzmoyev** [reply]
I guess it should say something like "Achievement unlocked" near the "16 stars" and "Bronze unlocked" near the "128 stars".

**Comment by aaronlamz** [comment]
Click the Delete button in the upper right menu.😄

**Reply by kuzmoyev** [reply]
There is no one there 😅

**Reply by aaronlamz** [reply]
It seems that it cannot be deleted. 
https://github.com/github-community/community/discussions/2948
